.travis.yml

```diff
@@ -6,7 +6,7 @@ sudo: required
 services: docker
 
 go:
-- 1.11.4
+- 1.12
 - tip
```

Comment:
I think this needs to be explicitly `1.12.2`

Reply:
Cool, thanks @hexfusion